pkg/loadtest/loader.go

```diff
@@ -216,7 +216,6 @@ func (t *Loader) streamRequests(in <-chan vegeta.Target) <-chan error {
 	go func() {
 		defer close(errs)
 		for tgt := range in {
-			tgt := tgt // pin
```

Comment:
why removed the pin?

Reply:
This issue in go that required variable pinning was resolved in golang 1.22
The new linter `copyloopvar` that replaced `exportloopref` enforces that

Reply:
🥳